src/sql/base/browser/ui/table/table.ts

```diff
@@ -405,4 +463,59 @@ export class Table<T extends Slick.SlickData> extends Widget implements IDisposa
 	public get container(): HTMLElement {
 		return this._tableContainer;
 	}
+
+	// Checks if the current table is a tree grid. Every tree grid should have a column with expandableColumnFormatter
+	private isTreeGrid(): boolean {
+		return this._grid.getColumns().filter(c => c.formatter === treeGridExpandableColumnFormatter).length > 0;
+	}
+
+	// Gets the index for the expandable column
+	private expandableColumnIndex(): number {
+		return this._grid.getColumns().findIndex(c => c.formatter === treeGridExpandableColumnFormatter);
```

Comment:
what does expandable column mean? in the changes above, I saw that you have code to expand/collapse a row when precess left/right keys in the first/last cell.

Reply:
It is the column which contains the chevron icon 

![image](https://user-images.githubusercontent.com/6816294/166810455-73ad0a0d-3a81-44b7-9a2f-7085e636e04d.png)